### DISCUSSION TITLE
Is there still interest in MusicXML?

### DISCUSSION THREAD

**Discussion by lemzwerg** [w3c-cg/musicxml#528] [General]
I wonder how many people are actually working on or interested in MusicXML.  Here, on github, I get no answer for almost all of the questions I ask or issues I report.  This is incredibly frustrating, and I wonder whether it makes sense at all to report stuff if it is not even acknowledged or confirmed.

It seems that current interest of the relevant W3.org people is focused entirely on MNX.  However, AFAICS, this project is still in its infancy, with zero support anywhere except a single, publicly available demo that shows some fundamental concepts but nothing more.

Until there is more progress I think that at least some energy should be diverted to maintain MusicXML.

**Comment by dspreadbury** [comment]
Please note that there is nobody at W3C itself working in our community group. Our community group is run under the auspices of the W3C, which provides a framework for intellectual property contribution and the infrastructure for running the project, but W3C community groups are not W3C working groups. Even so, most people who work on W3C projects are volunteers who are doing so in addition to their main occupations.

In the case of our community group, the three co-chairs all have demanding full-time roles, and we all devote as much time to the community group as is practical. The focus of our group is currently squarely on MNX, because our highest priority is to bring MNX to version 1.0 as quickly as possible.

MusicXML is mature and although we do anticipate updating the MusicXML specification further, we consider it to be more or less in maintenance mode, and the changes we make will be modest, maintain backwards compatibility, and ideally serve to clarify existing functionality in the specification rather than greatly expand it.

@mscuthbert is the spec editor for MusicXML and he will triage the issues that you have created in due course.

**Reply by lemzwerg** [reply]
OK, thanks for the reply and clarification.

**Comment by mscuthbert** [comment]
Hi @lemzwerg -- most of the work right now is getting the tooling situation set back up for making new releases of musicxml using the new documentation and DTD locations (yes, they will be moving).  Once that's all set, more point-by-point decisions will be made.  But it's moot until I'm fully comfortable making a new release without everything being f'd up...

**Reply by lemzwerg** [reply]
Thanks for the information.  Is your work happening in a repository so that other people can follow the progress of your work?

**Comment by JerryLan01** [comment]
If your question is about whether MusicXML is needed and important, look no further than the recent Finale development stoppage and the subsequent move to Dorico.  The ONLY way to move existing works to Dorico is by using MusicXML.  Although I have several things I need to move in, my interest in MusicXML is more as a scripting language. I have complex things in my head I wish to realize and feel the need to automate the process of moving them to realization, rather than writing them note by note.
